### PR TITLE
chore(android): bump sdk to 11.14.0

### PR DESCRIPTION
**Pull request by abdelhamid-f-nasser**
## Description of the change

Bump Android SDK to 11.14.0

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> [MOB-12990](https://instabug.atlassian.net/browse/MOB-12990) -- subtask of --> [MOB-12984](https://instabug.atlassian.net/browse/MOB-12984)

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] Issue from task tracker has a link to this pull request

[MOB-12990]: https://instabug.atlassian.net/browse/MOB-12990?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[MOB-12984]: https://instabug.atlassian.net/browse/MOB-12984?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,10 @@
 - Deprecate the old `StringKey.discardAlertCancel` and `StringKey.discardAlertAction` string keys for overriding the discard alert buttons as they had incosistent behavior between iOS and Android ([#1001](https://github.com/Instabug/Instabug-React-Native/pull/1001)).
 - Deprecate the old `StringKey.reproStepsListItemNumberingTitle` string key for overriding the repro steps list item (screen) title as it had incosistent behavior between iOS and Android ([#1002](https://github.com/Instabug/Instabug-React-Native/pull/1002)).
 
+### Changed
+
+- Bump Insabug Android SDK to 11.14.0 ([#1019](https://github.com/Instabug/Instabug-React-Native/pull/1019)) [See release notes](https://github.com/Instabug/Instabug-Android/releases/tag/v11.14.0)
```
Comment (a7medev) [inline CHANGELOG.md:19]: There's a typo here in `Insabug`.

**Review by a7medev — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (abdelhamid-f-nasser) [reply CHANGELOG.md:19]: Pardon me for missing this, and great catch as always 🔬

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,10 @@
 - Deprecate the old `StringKey.discardAlertCancel` and `StringKey.discardAlertAction` string keys for overriding the discard alert buttons as they had incosistent behavior between iOS and Android ([#1001](https://github.com/Instabug/Instabug-React-Native/pull/1001)).
 - Deprecate the old `StringKey.reproStepsListItemNumberingTitle` string key for overriding the repro steps list item (screen) title as it had incosistent behavior between iOS and Android ([#1002](https://github.com/Instabug/Instabug-React-Native/pull/1002)).
 
+### Changed
+
+- Bump Instabug Android SDK to 11.14.0 ([#1019](https://github.com/Instabug/Instabug-React-Native/pull/1019)) [See release notes](https://github.com/Instabug/Instabug-Android/releases/tag/v11.14.0)
+
```
Comment (a7medev) [inline CHANGELOG.md:20]: _(Same changes as the iOS PR)_
Let's move the "Changed" section right after "Added" as our Changelog Guidelines document states.

Also, I have a small suggestion to align our Changelog formatting 🙏🏼
```suggestion
### Changed

- Bump Instabug Android SDK to v11.14.0 ([#1019](https://github.com/Instabug/Instabug-React-Native/pull/1019)). [See release notes](https://github.com/Instabug/Instabug-Android/releases/tag/v11.14.0).
```

**Review by a7medev — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (abdelhamid-f-nasser) [reply CHANGELOG.md]: Thanks again for spotting this @a7medev , I've made the suggested changes, please re-check and let me know if there is any other adjustments.

**Pull request merged**